setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+ Install aws client
```

Comment:
Oops - typo in copy over. You'll need to comment this out, or it'll complain (ie. `Install: aws: No such file or directory`). You could also probs eliminate L3, L7, and L12 as they're redundant based on the echoes.

Reply:
weird I literally just did `mv` from the components directory? Why did it play me like that 💔

Reply:
Software just likes to see us suffer, through a hundred tiny cuts to our ego. 😢 💔